scripts/update/2z-job-loss-by-industry-by-state-update.R

```diff
@@ -54,25 +58,27 @@ state_job_loss <- state_unemp %>%
 assert(nrow(state_job_loss) == 51)
 
 ##----Create state job loss by industry figures-------------------------
-# Calculate the national job loss percentage to compare to states
-national_job_loss_calc <- national_industry_est %>%
-  summarise(tot_emp = sum(total_employment), 
-            unemp_tot = sum(unemployment_totals)) %>%
-  mutate(percent_change_employment = -unemp_tot / tot_emp) %>%
+
+# Calculate the factor to adjust state data to national job loss
+national_job_loss_factor <- state_job_loss %>%
+  summarise(unemployment_totals = sum(unemployment_totals),
+            total_employment = sum(total_employment)) %>%
+  mutate(percent_change_employment = -unemployment_totals / total_employment) %>%
   select(percent_change_employment) %>%
   pull()
 
-# Calculate the factor to adjust WA, NY, etc. to national job loss
 if (!is_bls){
-  national_job_loss_factor <- state_job_loss %>%
-    summarise(unemployment_totals = sum(unemployment_totals),
-              total_employment = sum(total_employment)) %>%
-    mutate(percent_change_employment = -unemployment_totals / total_employment) %>%
+  # Calculate the national job loss percentage to compare to states
+  national_job_loss_calc <- national_industry_est %>%
+    summarise(tot_emp = sum(total_employment), 
```

Comment:
This is throwing me an error because when I run the code, national_industry_est has the following columns:`
`reference` (net job loss in reference month) `latest` (net job loss in latest month), `percent_change_employment`, and `lodes_var`

Was script 2b or the file `job_change_all_states_most_recent.csv` changed somewhere else to add the `unemployment_totals` and `employment_totals columns`?